packages/design-system/src/components/web-components/ds-month-picker/ds-month-picker.tsx

```diff
@@ -0,0 +1,58 @@
+import type * as React from 'react';
+import { define } from '../preactement/define';
+import { MonthPicker, MonthPickerProps } from '../../MonthPicker';
+import { parseBooleanAttr } from '../wrapperUtils';
+
+const attributes = [
+  'button-variation',
+  'class-name',
+  'clear-all-text',
+  'default-selected-months',
+  'disabled-months',
+  'error-id',
+  'error-message-class-name',
+  'error-message',
+  'error-placement',
+  'hint-id',
+  'hint',
+  'inversed',
+  'label-class-name',
+  'label-id',
+  'label',
+  'name',
+  'requirement-label',
+  'root-id',
+  'select-all-text',
+  'selected-months',
+] as const;
+
+/* eslint-disable @typescript-eslint/no-namespace */
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-month-picker': React.DetailedHTMLProps<
+        React.HTMLAttributes<HTMLElement> & {
+          [K in (typeof attributes)[number]]?: string;
+        },
+        HTMLElement
+      >;
+    }
+  }
+}
+/* eslint-enable */
+
+interface WrapperProps extends Omit<MonthPickerProps, 'inversed'> {
+  inversed?: string;
+  rootId?: string;
+}
+
+const Wrapper = ({ children, rootId, ...otherProps }: WrapperProps) => (
+  <MonthPicker {...otherProps} id={rootId} inversed={parseBooleanAttr(otherProps.inversed)}>
```

Comment:
I think we should give the three array attributes the same treatment that we do the `options` array attribute in `ds-dropdown` where teams are allowed to directly assign arrays to the properties (`monthPicker['disabled-months'] = [1]`) or assign a JSON string in the actual HTML attribute. There's an angular team who requested that functionality on the `ds-dropdown`, so I think they'd probably want a similar thing here.

```suggestion
  <MonthPicker
    {...otherProps}
    id={rootId}
    inversed={parseBooleanAttr(otherProps.inversed)}
    disabledMonths={typeof disabledMonths === 'string' ? JSON.parse(disabledMonths) : disabledMonths}
 >
```

Could turn this into one of the `wrapperUtils`